firmware/firmware.ino

```diff
@@ -383,13 +421,48 @@ void loop()
   input_buffer[bytes_read] = '\0';
   
   // Copy incoming message to debug console.
-  Serial.print("<- ");
-  Serial.println(input_buffer);
-    
+  //Serial.print("<- ");
+  //Serial.println(input_buffer);
+  
   // Attempt to parse command
   handleCommand(input_buffer);
 }
 
+void batteryUpdateLoop()
+{
+  int rawVoltage = analogRead(board::V_BATT);
+  double voltageReading = 0.03516*rawVoltage+0.05135;
+
+  char output_str[128];
+  snprintf(output_str, 128,
+           "{"
+           "\"s4\":{"
+           "\"type\":\"battery\","
+           "\"data\":\"%.3f %f %f\""
+           "}"
+           "}",
+           voltageReading, 
+           platypus::motors[0]->current(),
+           platypus::motors[1]->current()
+          );
+  send(output_str);  
+  delay(1000);
+}
+
+/*void tempLoop(){
+  delay(500);
+
+  
+  //assumes es2 on port 3 and Do probe on port 2
+  double newTemp = ((platypus::ES2*)platypus::sensors[3])->getLastTemp();
+  if (newTemp != lastTemp){
+    //send updated temp to do probe
+    ((platypus::AtlasSensor*)platypus::sensors[2])->setTemp(newTemp);
+    lastTemp = newTemp;
+    Serial.println(newTemp);
+  }
+}*/
```

Comment:
If we use this loop sometimes, we should uncomment it, but not call it.
If we don't use this loop, we should just remove it.